src/cli.ts

```diff
@@ -1,5 +1,6 @@
 import tidy from './index';
-import { readFileSync, writeFileSync } from 'fs';
+import {dash} from 'rw';
+const {readFileSync, writeFileSync} = dash;
```

Comment:
Could you move the const below the imports

Reply:
Yes, solved!